microsoftgraph/microsoft-graph-explorer-v2
# Feat/disable auth tryit

## Overview

1. Hides the sign in button in the try it experience
2. Shows an informational message to users, asking them to use the complete Graph Explorer for  PUT, PATCH, POST and DELETE requests

### Demo

![graph explorer try it](https://user-images.githubusercontent.com/12011447/60952761-8c446500-a304-11e9-9de0-fabe1620c50e.JPG)

### Notes

Checks for the locale in the query strings to detect which mode Graph Explorer should run in. The locale is present only when Graph Explorer is launched in the **Try It** experience

## Testing Instructions

* Launch graph explorer locally
* Add the following params to the URL locale=en-us
* Observe that an informational banner is show and the Sign In button is not rendered
